syft/execution/communication.py

```diff
@@ -14,7 +14,9 @@
     "mid_get",
     "move",
     "remote_get",
-    "remote_send",
+    "remote_copy_to",
+    "move_and_point",
```

Comment:
For consistency, seems like this should maybe be `remote_move_and_point`? 🤔

Reply:
I was definitely not found of `move_and_point ` ^^
Do you think we can find a better naming completely?

Reply:
agreed - remote_move_and_point is better but it's still a little hard to grok. Not a deal breaker though.